stable_baselines3/a2c/a2c.py

```diff
@@ -4,13 +4,13 @@
 from typing import Type, Union, Callable, Optional, Dict, Any
 
 from stable_baselines3.common import logger
+from stable_baselines3.common.base_class import OnPolicyRLModel
 from stable_baselines3.common.type_aliases import GymEnv, MaybeCallback
 from stable_baselines3.common.utils import explained_variance
-from stable_baselines3.ppo.policies import PPOPolicy
-from stable_baselines3.ppo.ppo import PPO
+from stable_baselines3.common.policies import OnlineActorCriticPolicy
 
 
-class A2C(PPO):
+class A2C(OnPolicyRLModel):
```

Comment:
Didn't we say we wanted to rename it to `BaseAlgorithm` and `BasePolicy` to `BaseModel` ?

Reply:
Hmm I might have missed this one, but indeed "Algorithm" is better name for the algorithms. Perhaps the acting parts (class handling the network) should still be called a "Policy", as it does the acting/stepping? "Model" sounds more like just the inner-network part, that outputs some stuff and does not care what it does outside that.